packages/flame_console/example/pubspec.yaml

```
@@ -0,0 +1,24 @@
+name: example
+description: "Example of using the flame console package"
+
+publish_to: 'none'
+
+version: 1.0.0+1
+
+environment:
+  sdk: ">=3.4.0 <4.0.0"
+
+dependencies:
+  flame: ^1.19.0
+  flame_console:
+    path: ../
```

```suggestion
  flame_console: ^0.1.0
```
Doesn't matter that it's not released, melos bootstrap will set it up.